docs/notebooks/mk_data_exploration_pep725.ipynb

```diff
@@ -0,0 +1,2571 @@
Emv243l6/356BmjSjWPieY8xhPtUGek9UK6QzUgJNXFhkbRjq5+mJEGgMHF/g4ZdXu1Qrpd1y88XiJNxzQi6EhTSBNCCEkPFNIkqUwfXgK7xYy65i68tHoHAG0+GkhuRlodqWjVRzvc6mhH6vpIA4DD4OTAKaMd1v71MxtcFBDSzXoh7f/8E5lqCOgG20QR0s2dLuXEaWgJox2EEELSQ/86wpO0k2O3Yv/aYgD+DhoWswkHjRykeYzSR9qTjIx0sFtGOEdaCKEUHqaq2NCw/V2KXzNdVBcZO9JygqUcBR8vmjHzUTp3SDe6siCr332+hBBCMgcKaZJ01A70lJoiZVCLxB5lsqFPyPZ30TfPWIoNu90+BBaZumiH4UCW/h3t2KFzpNu6e9a1QzNmPoojHezYQTeaEEJI+qCQJklH9pPW/78kWkZaGcgSw9YZS7Fhp2rKYXaKRK1h+7tAtKO/DQsJF+3oabEhoJ5uGLnYcPNu5qMJIYSkHwppknTGVxWgosABADhiTHnI/TIjbTR4w+cTinscmyMdvthQCmgpqB1Wc0xxkURw6CYsAkFHvD917QCAIWGKDWWv7p4I6Vhb4ElHeigdaUIIIWkkNde5yYDGbDbhH2cfgO17uzClpijk/kjt77wi6ETGO5AlnCMtiw5TVWioXg/nAIh2yIx0m9ODli63Et3p6WRDIPbphpsDQpqONCGEkHTSv6wy0meYOLgQx0ysNLxPuo4+gZB+weq/Ex3IInO2wWhHalvfAcYCUHbtSFWcJF3k2K0oybUD0LrSPZ1sCETvMS7ZEig2ZEaaEEJIOqGQJr2OTdXmTC+YPCohnehAlorAwJBgtCO1re8AwG7xL1sdSejup32kAeOcdE8nGwKq2E8EId3a7caeDv8IegppQggh6YRCmvQ66sEbesEUvyMdjFTIPtJVhVJIBxxpZ+qjHUr7O7fRQJb+9zNThPRevzNc39KNTpcXZhNQHsjHJ0Kw2DC8kN4acKNL8+zIz7KFfRwhhBCSapiRJr1OpH7BGiFtikFI24JC2hcQ0pUBIS2L/VLdQxpQtb/z+l9LCNG/HelirSO9etteAMA+Ffk9itDEUmy4maPBCSGE9BEopEmvI/sFe3wCbq82I+3x+QWUyeR/XDSkgHV6vOh0+Zcli+FkF4ngVMPey0i7vQLynCCrn7W/A4KO9I7mbgDAF9uaAQBThxb1aLmxFBsyH00IIaSvQCFN0oLNYobH5w3JSAd0dEz5aABwBNzeTpdXcbMrC3TRDlfqHWmHTgB2qyYcZtn7X7RDnqxsl4701mYAwNSa4h4tV5l66Q3fR3oLO3YQQgjpI/S/IzzJCMIVlUlH2hxDrAMIOtLqSEhItCPw39xeENKy/Z2MdZhMwXXsT6h7SXu8PnxV1wIAmNJTR1oXkTFiMx1pQgghfYT+d4QnGUG4MeFSEMfuSGs34XyHVeka0SH7SKeh/V23y//fLKsFphhPCjIJGe3Y3e7EV3Ut6HR5keewYmRZXo+WG8tkQzrShBBC+goDSkj/5z//SfcqkADKJXyPcR/pWCcQ6t3egmyb0p1DZqNlVjqlxYZWbZFccBhL//yJFeUEP+fXv64HAOw7pLDHkyOjtb/rdHnQ0OoEQCFNCCEk/fTPo3wYfvrTn2LRokVob29P96oMeMJ1Z4hXSDus2k24MNumOM8hjnQKu2eoB8MA/buHNACYTCbFlX7ty50Ael5oCKi2izDFhlub/LGOohwbCnPY+o4QQkh6GVBCeuXKlfj000+x7777Yvny5elenQGNdB7DDWSxmGPbNE0mk+IGA1qnVJ+R7o32d0Eh3T+nGqrRt8Cb0sNCQyB85EeyebfMR9ONJoQQkn4GlJCePHkyPv30UyxYsABHH300Lr/8cjQ1NaG1tVXzj6QeW5hR0PFmpAHAoYp3FKqiHZ0uD4QQaRkRLh1pR38W0gFHWjKlpqjHy7RHcaRlPnpYCQsNCSGEpJ8B1/7OarXi+uuvx8EHH4xjjz0Wd999t3KfEAImkwneCB0DSHKIVmwYT9bWYTOjzR+b9QvpQLGhT/gd4i63PyOd6+iN9neBOEk/nmookY404O/iUZaf+ERDSVRHOtCxo5YdOwghhPQBBpyQBoDnn38ev/zlL3HYYYfhd7/7HazWAfkxpJVgFlY/kCV+IW3XOdLqOEWny4uOwIjwVMYs7GHa3/XHYSwStSOdDDcaUGfnjbt2bOFUQ0IIIX2IAaUgm5ubceGFF+Kll17CzTffjN/85jfpXqUBS7iMtE8kEO1QCeSCbBssZhMcVjOcHh86XZ60tL9zyox0CnPZ6SalQjpstCPgSJfSkSaEEJJ+BpSQHj9+PIYOHYrPP/8cY8aMSffqDGjCZaQ9AScylvHgEr0jDQC5DiucHhc6XV50ulPf/s4xwNrfAdpoRzI6dgCRox3dbi92tPgLG+lIE0II6Qv036O8ARdeeCE++ugjiug+gD2ZxYa2UCGdrRodHnSkU9/+Tjqp8jX7c7SjPD8LQ0tyUFHgwITqwqQs0y77SBs40tv3dkIIIM9hxaBce1JejxBCCOkJA8qR/v3vf6/5e9WqVfjuu+8A+N3q/fbbLx2rNSAJl4X1igSKDa2hQlrduaOzF4R0aEba/9/+3LXDYjbh9d8cCoHk9cuO5EhvUY0G74/TIgkhhGQeA0pISxobGzF//ny8//77KCoqAuDPTx955JF48sknUVZWlt4VHADYlFHQekfa/3dcxYZGQlqOCXd6lQ4avdFH2usT8PrEgIh2AP4ITTIJN6gHUHfsYKyDEEJI36B/H+XD8Otf/xptbW345ptv0NTUhKamJnz99ddobW3FxRdfHPNyli9fjnnz5qG6uhomkwkvvvii5n4hBK677jpUVVUhOzsbs2bNwvr16zWPaWpqwplnnomCggIUFRXhvPPOGxCTF8MOZPEm4kgHBbIipAMO6d4OFwImN3J7odgQ8McSZNeO/jyQJRVEKjYMduxgoSEhh
```

Comment:
While the code looks okay, it seems impractical to have to do this all the time. It might be more convenient if we could do something like:

```
dataset = PEP725Phenor(
    dataset = 'PEP725Phenor', 
    species = 'Syringa vulgaris', 
    area = Bbox(name='Enschede', center=[52.2215372, 6.8936619], radius_km=25), 
    ...
)
```

Is this something you would welcome? Then we could open an issue and see if we can prioritize it.